pyuvdata/uvdata/uvdata.py

```diff
@@ -2460,6 +2460,206 @@ def reorder_blts(
                 strict_uvw_antpos_check=strict_uvw_antpos_check,
             )
 
+    def reorder_freqs(
+        self,
+        spw_order="number",
+        channel_order=None,
+        select_spw=None,
+        run_check=True,
+        check_extra=True,
+        run_check_acceptability=True,
+        strict_uvw_antpos_check=False,
+    ):
+        """
+        Arrange frequency axis according to desired order.
+
+        Can be applied across the entire frequency axis, or just a subset.
+
+        Parameters
+        ----------
+        spw_order : str or array_like of int
+            A string describing the desired order of spectral windows along the
+            frequecy axis. Allowed strings include `number` (sort on spectral window
+            number) and `freq` (sort on median frequency). A '-' can be appended
+            to signify descending order instead of the default ascending order,
+            e.g., if you have SPW #1 and 2, and wanted them ordered as [2, 1],
+            you would specify `-number`. Alternatively, one can supply an index array
+            of length Nspws that specifies the new order. Default is to apply no
+            sorting of spectral windows.
```

Comment:
This last comment confuses me, it looks like the default is `spw_order="number"`, which I believe means it will by default sort by spws.